packages/@o3r/core/schematics/rule-factories/differential-loading/index.ts

```diff
@@ -60,7 +60,7 @@ export function updateDifferentialLoading(options?: { projectName: string | null
       }
 
     } else {
-      context.logger.warn('The tsconfig.prod.json file has not been created');
+      context.logger.warn('Diferential loading setup skipped. The tsconfig.prod.json file has not been created');
```

Comment:
is differential loading still relevant now that we don't support IE11 anymore?

Reply:
Nice catch :)
it's not relevant. I will remove it from the ng add.